### DISCUSSION TITLE
Is there a way to deactivate some of the restFul ednpoints? Or how do i protect them?

### DISCUSSION THREAD

**Discussion by andreiSeer** [dcm4che/dcm4chee-arc-light#3448] [General]
Starting now with dcm4chee and i was wondering if it has a way to protect or deactivate some of those endpoints.

**Comment by vrindanayak** [comment]
@andreiSeer : Your use case is not clear. Can you explain it a bit more?

If you are using archive UI and want to restrict functions for certain users, you may configure [UI Permissions](https://github.com/dcm4che/dcm4chee-arc-light/wiki/UI-Permission-Configuration)

Additionally, users of any client accessing one of [DICOM Web Services](http://dicom.nema.org/medical/dicom/current/output/html/part18.html) of archive secured with Keycloak can be restricted based on the roles assigned to them in Keycloak - implemented since version 5.21.0 in this [issue](https://github.com/dcm4che/dcm4chee-arc-light/issues/2400).
